internal/scaffold/ansible/roles_meta_main.go

```diff
@@ -97,4 +97,6 @@ const rolesMetaMainAnsibleTmpl = `galaxy_info:
 
 dependencies: []
   # List your role dependencies here, one per line. Be sure to remove the '[]' above,
-  # if you add dependencies to this list.`
+  # if you add dependencies to this list.
+collections:
+- operator_sdk.util`
```

Comment:
Is there a best practice between declaring the collection like this and then using the shortened name in tasks vs using the fully qualified name in tasks?

Reply:
They don't seem to have a preference yet.
https://docs.ansible.com/ansible/devel/user_guide/collections_using.html#using-collections-in-a-playbook